File: 1/b.py
target = 2020
result = 0
numbers = []
with open("input.txt") as f:
    numbers = f.read().splitlines()

for i in range(len(numbers)):
    for j in range(len(numbers)):
        for k in range(len(numbers)):
            if int(numbers[i]) + int(numbers[j]) + int(numbers[k])== target:
                result = int(numbers[i]) * int(numbers[j]) * int(numbers[k])

print(result)

File: 1/b.c
#include <stdio.h>
#include <stdlib.h>
#include "../input.h"

int main() {
    int target = 2020;
    int result = 0;

    input_t *input = InputRead("input.txt");
    if (input == NULL) {
        return EXIT_FAILURE;
    }

    for (int i = 0; i < input->length; ++i) {
        for (int j = 0; j < input->length; ++j) {
            for (int k = 0; k < input->length; ++k) {
                int n1 = atoi(input->arr[i]);
                int n2 = atoi(input->arr[j]);
                int n3 = atoi(input->arr[k]);
                if (n1 + n2 + n3 == target)
                    result = n1 * n2 * n3;
            }
        }
    }

    printf("%d\n", result);

    InputFree(input);
    return EXIT_SUCCESS;
}
